HelloDroid/Resources/layout/activity_dynamic.xml

```diff
@@ -0,0 +1,89 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/activity_dynamyc_relativeLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center_vertical">
+
+    <TextView
+        android:id="@+id/activity_dynamyc_textView_switch"
+        android:text="Text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        style="@style/TextViewPrimary"
+        android:textSize="20sp"
+        />
+
+    <Switch
+        android:id="@+id/activity_dynamyc_switch"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/activity_dynamyc_textView_switch"
+        android:layout_centerHorizontal="true"
+        />
+
+    <TextView
+        android:id="@+id/activity_dynamyc_textView_counter"
+        android:text="Text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/activity_dynamyc_switch"
+        android:layout_marginTop="50dp"
+        android:layout_centerHorizontal="true"
+        style="@style/TextViewPrimary"
+        android:textSize="20sp"
+        />
+
+    <LinearLayout
+        android:id="@+id/activity_dynamyc_linearLayout"
+        android:orientation="horizontal"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/activity_dynamyc_textView_counter"
+        android:layout_centerHorizontal="true">
+
+        <Button
```

Comment:
Buttons are inside a container (LinearLayout) - they need to be positioned only relatively to their closest parent